modules/zrtp/zrtp.c

```diff
@@ -112,6 +127,31 @@ static void media_destructor(void *arg)
 }
 
 
+static void abort_timer_h(void *arg)
+{
+	const struct menc_sess *sess = arg;
+
+	if (sess->errorh) {
+		sess->errorh(sess->err, sess->arg);
+	}
```

Comment:
perhaps also set `errorh` to `NULL` so that the callback handler is only called once.